OracleSOABPM/dockerfiles/README.md

```diff
@@ -0,0 +1,33 @@
+How to Build the image
+======================
+Before you Start
+----------------
+1. **FIRST TIME ONLY**
+   - Update the OracleSOABPM/setenv.sh file to point
+     to the right locations for the following environment variables
+     specific to your environment (See top of file)    
+     DC_USERHOME, DC_REGISTRY_SOA, DC_REGISTRY_DB    
+   - Update the OracleSOABPM/setenv.sh file with the proxy details
+     if you have an internal proxy - ensure that the proxy
+     related env variables are set    
+     http_proxy, https_proxy, no_proxy
+   - Pull the base image into your local docker cache for builds.
+     You will need to provide your login credentials.
```

Comment:
This is more an internal thing here - though users may face it too. Basically the proxy values not being set results in very slow downloads as it contacts the proxy for internal URLs too. This is again generic stuff that we found along the way as we were doing work - and documented it for others that may run into the same issue.